docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -299,6 +299,14 @@ The ability to create a project for an earlier TFM depends on having that versio
 
   Doesn't execute an implicit restore during project creation.
 
+- **`--kestrelHttpPort`**
+
+  Port number to use for the HTTP endpoint in *launchSettings.json*.
+
+- **`--kestrelHttpsPort`**
+
+  Port number to use for the HTTPS endpoint in *launchSettings.json*. This option is not applicable when the parameter `no-https` is used (but `no-https` is ignored when `IndividualAuth` or `OrganizationalAuth` is used).
```

Comment:
I don't see `IndividualAuth` and `OrganizationalAuth` listed as `auth` options - https://review.docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new-sdk-templates?branch=pr-en-us-28394#web-options?

Reply:
Good catch, thanks for the review!